src/parser/SparqlParserHelpers.cpp

```diff
@@ -19,6 +19,7 @@ ParserAndVisitor::ParserAndVisitor(
   // propagate them to the user.
   parser_.removeErrorListeners();
   parser_.addErrorListener(&errorListener_);
+  visitor_.setStartTime();
```

Comment:
Such manual calls are always a bit prone to being forgotten.
Is there anything that speaks against calling `setStartTime()` automatically in the constructor of the visitor?